package.json

```diff
@@ -177,6 +178,7 @@
     "inherits",
     "ini",
     "init-package-json",
+    "is-ci",
```

Comment:
Awesome! You added it to `bundleDependencies`. Additionally, we commit our dependencies, and we use `package-lock.json` so please go ahead and `git add -A node_modules/ package-lock.json` as well before you commit the `ci-info` change I asked for above!